app/models/data_source_adapters.rb

```diff
@@ -0,0 +1,19 @@
+module DataSourceAdapters
+  class << self
```

Comment:
I'm using "def self.adapters; ..." style (not "class << self; def ..." style) in this project, so I want you to arrange like that.